tests/src/test/java/alluxio/server/tieredstore/SpecificTierWriteIntegrationTest.java

```diff
@@ -45,8 +45,8 @@
  * Integration tests for writing to various storage tiers.
  */
 public class SpecificTierWriteIntegrationTest extends BaseIntegrationTest {
-  private static final int CAPACITY_BYTES = Constants.KB;
-  private static final int FILE_SIZE = CAPACITY_BYTES;
+  private static final int CAPACITY_BYTES = Constants.KB * 10;
+  private static final int FILE_SIZE = CAPACITY_BYTES / 10;
```

Comment:
similarly, the next person who comes along might not realize that the `10` on capacity and file size are related. Maybe make it a single constant variable and mult/divide by that?

Reply:
I've changed some of the naming to make the relationships more explicit.